flags/flag_names.go

```diff
@@ -41,14 +41,23 @@ const (
 	ValidatorChainConfigFileFlag    = "validator-chain-config"
 
 	TekuTagFlag                 = "teku-tag"
+	TekuDatadirFlag             = "teku-datadir"
 	TekuConfigFileFlag          = "teku-config"
 	TekuValidatorConfigFileFlag = "teku-validator-config"
-	MainnetFlag                 = "mainnet"
-	TestnetFlag                 = "testnet"
-	DevnetFlag                  = "devnet"
-	ValidatorFlag               = "validator"
-	ConsensusFlag               = "consensus"
-	ExecutionFlag               = "execution"
+
+	Nimbus2TagFlag                 = "nimbus2-tag"
+	Nimbus2CommitHashFlag          = "nimbus2-commit-hash"
+	Nimbus2NetworkFlag             = "nimbus2-network"
+	Nimbus2DatadirFlag             = "nimbus2-datadir"
+	Nimbus2ConfigFileFlag          = "nimbus2-config-file"
+	Nimbus2ValidatorConfigFileFlag = "nimbus2-validator-config-file"
+
+	MainnetFlag   = "mainnet"
+	TestnetFlag   = "testnet"
+	DevnetFlag    = "devnet"
```

Comment:
Do we need `devnet` flag still? Let's remove `devnet` from CLI in next PR :+1:

Reply:
:+1: